src/elements/select/SelectPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Icon from '../icon/Icon';
+
+// take javascript example from https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/listbox/listbox-collapsible.htmla
+
+function SelectPage() {
+  const themes = ['zeppelin', 'cat', 'rental'];
+  return (
+    <>
+      <p style={{ marginBottom: 45 }}>With List</p>
+      {themes.map(elem => (
+        <div
+          className={`theme-${elem}`}
+          style={{ marginBottom: 15 }}
+          key={elem}
+        >
+          <div className="zep-select">
+            <label id="exp_elem" className="zep-select__label">
+              Label
+            </label>
+            <div id="exp_wrapper">
+              <button
+                aria-haspopup="listbox"
+                aria-labelledby="exp_elem exp_button"
+                id="exp_button"
+                className="zep-select__button"
+              >
+                Select One
+                <Icon
+                  className="zep-select__icon"
+                  icon="zepicons-navigation-dropdown"
+                />
+              </button>
+              <ul
+                id="exp_elem_list"
+                tabIndex="-1"
+                role="listbox"
+                aria-labelledby="exp_elem"
+                className="zep-select__list"
+                //className="zep-visually-hidden"
+              >
+                <li
+                  id="exp_elem_A"
+                  role="option"
+                  tabIndex="0"
+                  className="zep-select__list__item"
```

Comment:
There are no grandchildren in BEM, it should be .zep-select__listitem or if we already have lists it could also be .zep-list__item--select to modify an existing list item class.